### PR TITLE
Correct wording

### PR DESCRIPTION
**Pull request by RussKie**
Relates to #51

### CODE REVIEW

**Review by gerhardol — APPROVED** [review]

File: src/PackageManager.UI/Views/MainWindow.xaml.cs [hunk 1]
```diff
@@ -53,8 +53,7 @@ private Task<bool> OnBeforeChange()
             {
                 bool result = navigator.Confirm(
                     "Plugin Manager",
-                    "Plugin Manager is going to write to files that are holded by other executables. " + Environment.NewLine +
-                    "Do you want to kill all instances of these applications?"
+                    "Plugin Manager will be writing to files that are currently in use.\r\n\r\nDo you want to stop all instances of the application?"
```
Comment (RussKie) [inline src/PackageManager.UI/Views/MainWindow.xaml.cs:56]: @maraf even in a situation when the PM is run outside GE, "instances of these applications" is very ambiguous. Like, which applications?
In GE case, it is "the application", and the use of the plural case also looks confusing.

Comment (mast-eu) [reply src/PackageManager.UI/Views/MainWindow.xaml.cs:56]: I agree with RussKie. I always found this message sounds too "technical", but could find a good replacement.

**Review by mast-eu — APPROVED** [review]

File: src/PackageManager.UI/Views/MainWindow.xaml.cs (hunk 1 above)
Comment (maraf) [reply src/PackageManager.UI/Views/MainWindow.xaml.cs:56]: I agree with changing the message. As commented on #51, we may use passed names of processes and display them to user.

**Comment by maraf** [comment]
Closed in favor of #54.

**Pull request closed without merging**